katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -0,0 +1,376 @@
+Name: katello-host-tools
```

Comment:
I assume all the code to support older versions like EL5 is still relevant in this case.

Reply:
Yup.